src/components/ResultsView.tsx

```diff
@@ -546,6 +613,23 @@ const ResultsView: ResultsViewComponent = ({
                       setIsEditViews(true);
                     }}
                   />
+                  <MenuItem
+                    icon={showInterface ? "th-disconnect" : "th"}
+                    text={showInterface ? "Hide Interface" : "Show Interface"}
+                    onClick={() => {
+                      const resultNode = getSubTree({
+                        key: "results",
+                        parentUid,
+                      });
+                      setInputSetting({
+                        key: "interface",
+                        value: showInterface ? "hide" : "show",
+                        blockUid: resultNode.uid,
+                      });
+                      setShowInterface((s) => !s);
+                      setMoreMenuOpen(false);
```

Comment:
oh wait, one last thing before we merge - now that this is sourced from `results.interface` we need to update the `useState` for `showInterface` to source from the correct block instead of layout

Reply:
Nice catch